src/xrCore/xrDebug.cpp

```diff
@@ -97,7 +97,7 @@ size_t xrDebug::BuildStackTrace(char* buffer, size_t capacity, size_t lineCapaci
 
 size_t xrDebug::BuildStackTrace(EXCEPTION_POINTERS* exPtrs, char* buffer, size_t capacity, size_t lineCapacity)
 {
-    memset(buffer, capacity * lineCapacity, 0);
+    memset(buffer, 0, capacity*lineCapacity);
```

Comment:
May be with this stack trace will be more appropriate, which is not in some cases(for example, just gives nothing)